pages/public_cloud/compute/transfer_instance_backup_from_one_datacentre_to_another/guide.en-gb.md

```diff
@@ -131,6 +148,11 @@ $ openstack image create --disk-format qcow2 --container-format bare --file snap
 
 ### Create an instance from your backup
 
+> [!warning]
+>
+> If your instance is a Windows server, you must select a flavor of type win-xx-xx (for example, win-b2-15) and have a public interface on the Ext-Net network. Without these conditions, authentication with the OVH KMS will not be possible, and your server will remain with an [unactivated licence](https://help.ovhcloud.com/csm/en-gb-public-cloud-compute-activate-windows-licence-private-mode-instance?id=kb_article_view&sysparm_article=KB0056164). This could lead to limitations, including the absence of updates. Please note that it is not possible to resize a Linux instance (such as b2-15) to a Windows instance (such as win-b2-15). To make this transition, you need to recreate a new instance.
```

Comment:
```suggestion
> If your instance is a Windows server, you must select a flavor of type win-xx-xx (for example, win-b2-15) and have a public interface on the Ext-Net network. Without these conditions, authentication with the OVHcloud KMS will not be possible, and your server will remain with an [unactivated licence](/pages/public_cloud/compute/activate-windows-license-private-mode). This could lead to limitations, including the absence of updates. Please note that it is not possible to resize a Linux instance (such as b2-15) to a Windows instance (such as win-b2-15). To make this transition, you need to recreate a new instance.
```